apps/site/src/pages/settings/billing-settings-panel/billing-page-context.tsx

```diff
@@ -0,0 +1,15 @@
+import { SubscriptionTierPrices } from "@local/internal-api-client";
+import { createContext, useContext } from "react";
+import Stripe from "stripe";
+
+export type BillingPageContextValue = {
+  paymentMethods?: Stripe.PaymentMethod[];
+  refetchPaymentMethods?: () => Promise<void>;
+  subscription?: Stripe.Subscription;
+  refetchSubscription?: () => Promise<void>;
+  subscriptionTierPrices?: SubscriptionTierPrices;
+};
+
+export const BillingPageContext = createContext<BillingPageContextValue>({});
+
+export const useBillingPageContext = () => useContext(BillingPageContext);
```

Comment:
I'm not what value this provides, but since it's more consistent I've addressed this in https://github.com/blockprotocol/blockprotocol/pull/933/commits/ac9bf9368079da9c833d68a5338f2026958ecba3

Reply:
Thank you, it suggested it for consistency sake to be honest.